### ISSUE TITLE
docs: Add writing test instructions

### ISSUE THREAD

**Issue by gnikit** [lfortran/lfortran#943]
It would be great if the Contributing section: https://docs.lfortran.org/en/contributing/ in the docs, also included instructions on

- How to write tests
- How to run the test suite to validate your changes

**Comment by certik** [comment]
Indeed! We have been evolving as well on this, at this point the main tests are `integration_tests`, those are the ultimate tests and if they run, it probably works.

**Comment by HarshitaKalani** [comment]
Hey, can you please suggest me how can I preview the .po file in vs code in macbook?

**Comment by certik** [comment]
@HarshitaKalani I don't know how to do that. If you figure it out, please share it, others will find it useful as well.
